voice/guides/cnamService.md

```diff
@@ -0,0 +1,44 @@
+# CNAM per DIP API setup instructions
+
+## Overview
+Bandwidth's Caller Name (CNAM) service is a low-cost per-query service that provides the Caller Name on VoIP calls in the US and Canada.
+With this service, the CNAM database is queried using a simple HTTPS GET Request.
+Once the request is made, the calling name is returned to the customer application or dial plan.
+The CNAM data is managed by the phone company who provides the phone numbers to its customers.
+Bandwidth can not guarantee the validity of the information returned by the managing phone company.
+
+To utilize the CNAM lookup API, you will need to be setup with an account by your Account Manager.
+Please open a [support ticket](https://support.bandwidth.com/hc/en-us/requests/new) to enable this feature for your account.
+
+## CNAM setup instructions via HTTPS
+The simplest method of accessing the CNAM service is via an HTTPS GET request.
+The query information will be passed in the GET, and the entire response will be the CNAM data.
+Using an HTTP library that supports keep-alives, as well as HTTP pipelining, can help ensure highest performance service.
+
+Bandwidth's CNAM service will authenticate based on two factors:
+   1. The `CompanyID` passed will be checked to ensure CNAM service is enabled.
+   1. A password will be validated.
+
+## Query format
+`https://cnam.dashcs.com/?companyId=<companyId>&password=<pass>&number=<phone number>`
+
+Note: Please ensure that the password is URL-encoded.
+
+## Query parameters 
+| Query Type                | Required or Optional | Type                     | Value/Notes |
+|---------------------------|----------------------|--------------------------|-------------|
+| CompanyId                 | Required             | Integer                  | Assigned during implementation/onboarding. |
+| Password                  | Optional             | String                   | Assigned during implementation/onboarding. |
+| Number                    | Required             | 11-digit TN              | (1+10 digit TN) that's used to query CNAM information. |
```

Comment:
should still be specific that Number and DNIS are `string` s

Reply:
Changed the way it was written.